README.md

```diff
@@ -3,40 +3,265 @@
 </a>
 
 # TwiML Generator
-A tool to help generate sample code for creating TwiML with Twilio's helper libraries
+A tool to help generate sample code for creating TwiML with Twilio's Helper Libraries
 
-## Supported Languages
+## Supported Helper Library Versions
 
-| Language  | SDK Version |
+| Helper Library  | Version |
 | :-------------  |:------------- |
-| C# | 5.x |
-| Java | 7.x |
-| Node | 3.x |
-| PHP | 5.x |
-| Python | 6.x |
+| [twilio-csharp](https://github.com/twilio/twilio-csharp/) | 5.x |
+| [twilio-java](https://github.com/twilio/twilio-java/) | 7.x |
+| [twilio-node](https://github.com/twilio/twilio-node/) | 3.x |
+| [twilio-php](https://github.com/twilio/twilio-php/) | 5.x |
+| [twilio-python](https://github.com/twilio/twilio-python/) | 6.x |
+| [twilio-ruby](https://github.com/twilio/twilio-ruby/) | ? |
 
-## Using the tool
 
-You can use `./generator.py` to print out and save code that is generated from a TwiML file:
+## Requirements
+The generator tool will try to test the generated snippets using a local
+environment for every language's SDK version.
+
+In order to use the testing functionality, you need to install each Helper Library. 
+
+### Helper Library Installation
+<details>
+  <summary>Click to expand</summary>
```

Comment:
Maybe this could have a bit more detail, like "Click to see installation instructions for each helper library"?